### ISSUE TITLE
CSV Import fails with models and categories

### ISSUE THREAD

**Issue by maximilianvw** [snipe/snipe-it#2418]
#### Expected Behavior (or desired behavior if a feature request)

Importing assets from a CSV File. Examples from the following website are used.
https://snipe-it-manual.readme.io/docs/importing-assets

---
#### Actual Behavior

Most fields like User, Asset Tag or Purchase Date are imported correctly but values like the models or categories will be just "Unknown" or "Unknown Category".

![screen shot 2016-08-09 at 16 32 56](https://cloud.githubusercontent.com/assets/18569670/17520180/0786014a-5e4f-11e6-8780-d5a1a877e5f7.png)

Fun part is that if i force some kind of error while importing, the debug log will show that the values are mapped to the right field:

![screenshot](https://cloud.githubusercontent.com/assets/18569670/17547015/20c515a6-5ee5-11e6-914c-83f43f924538.png)
In this case I added an extra comma after the asset serial but you can clearly see that model name and the category is recognized.

Same behavior when tested the Import with just one asset from the example CSV files you provided in the docs...

What am i doing wrong? :(

---
#### Please confirm you have done the following before posting your bug report:
- [x] I have enabled debug mode
- [x] I have read [checked the Common Issues page](http://docs.snipeitapp.com/common-issues.html)

---
#### Please provide answers to these questions before posting your bug report:
- Version of Snipe-IT you're running 
  3.2.0
- What OS and web server you're running Snipe-IT on 
  Ubuntu 14.04, Apache 2.4
- What method you used to install Snipe-IT (install.sh, manual installation, docker, etc)
  install.sh
- If you're getting an error in your browser, include that error
- What specific Snipe-IT page you're on, and what specific element you're interacting with to trigger the error
  hardware/import
- If a stacktrace is provided in the error, include that too.
- Any errors that appear in your browser's error console.
- Confirm whether the error is [reproduceable on the demo](https://snipeitapp.com/demo).
  Import is disabled in demo.
- Include any additional information you can find in `app/storage/logs` and your webserver's logs.
- Include what you've done so far in the installation, and if you got any error messages along the way.
- Indicate whether or not you've manually edited any data directly in the database

**Comment by dmeltzer** [comment]
Hi,

I think this has been fixed in the latest `develop` branch code.  We adjusted a lot of formatting there.

Something does look weird though, it seems to think that the purchase_cost field for this asset is set to `19.12.2012`, which looks much more like a purchase_date to me.  Can you confirm that there are no errors in the csv?

**Comment by mrabbat** [comment]
I am facing this same problem and I'm running version 3.4. Model is always showing as unknown ! 
Any fix for that ?

**Comment by ctober** [comment]
I have discovered that if you export a file (csv, excel, etc). The model field will be titled "Model" and NOT "Model Name". If you attempt to make edits and re-import the csv file with the model column still titled "Model" it will not throw an error but will show the "Model Name" as Unknown. This is how I managed to produce the problem in my environment, I would guess you might have tried something similar. It may be worth requesting that all instances of "Model" get changed to "Model Name" unless this has side effects I am unaware of.
